### PR TITLE
close file fd

### PR DESCRIPTION
**Pull request by jiuker**
**Please provide a description of this PR:**

close file fd

**To help us figure out who should review this PR, please put an X in all the areas that this PR affects.**

- [ ] Ambient
- [ ] Configuration Infrastructure
- [ ] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [x] Developer Infrastructure

**Please check any characteristics that apply to this pull request.**

- [ ] Does not have any [user-facing](https://github.com/istio/istio/tree/master/releasenotes#when-to-add-release-notes) changes. This may include CLI changes, API changes, behavior changes, performance improvements, etc.

### CODE REVIEW

File: pkg/bootstrap/config.go [hunk 1]
```diff
@@ -381,6 +381,7 @@ func getProxyConfigOptions(metadata *model.BootstrapNodeMetadata) ([]option.Inst
 			if err != nil {
 				return nil, err
 			}
+			defer lsConfigOut.Close()
```
Comment (howardjohn) [inline pkg/bootstrap/config.go:384]: should we just use something like os.Writefile

**Comment by jiuker** [comment]
@howardjohn review plz

**Review by howardjohn — APPROVED** [review]

**Pull request merged**